easybuild/base/testing.py

```diff
@@ -124,6 +124,11 @@ def assertNotExists(self, path, msg=None):
             msg = "'%s' should not exist" % path
         self.assertFalse(os.path.exists(path), msg)
 
+    def assertAllExist(self, paths, msg=None):
+        """Assert that all paths in the given list exist"""
+        for p in paths:
+            self.assertExists(p, msg)
```

Comment:
@gkaf89 Style nitpicking: please avoid using single-letter variables outside of list comprehensions (I see this popping up in a couple of places in this PR):

```suggestion
        for path in paths:
            self.assertExists(path, msg)

Reply:
Fixed single character variable names and other minor style issues in a separate commit.